text/0055-signature-based-finality.md

```diff
@@ -32,43 +32,42 @@ The current process of block finalization is this:
 5. Consensus creates a finality signature for the block.
 6. Linear chain stores the signature and gossips it.
 
-In order to only consider fully signed blocks as finalized, we'll have to change this process to the following:
+We wouldn't change the meaning of "finalized" in order not to introduce confusion - a block being finalized would still mean finalized by consensus. However, we would add a new, "signed" status of the block, which would be used instead of "finalized" for some purposes (for example, advancing to the next era will require the switch block to be signed, instead of only finalized). The process would then look the following way:
```

Comment:
Maybe add that the meaning of the "signed" status is that we have a _concise_ proof of finality. (As opposed to the complicated, large one, that would be a Highway summit.)